client/src/main/java/com/netflix/conductor/client/http/TaskClient.java

```diff
@@ -225,6 +225,7 @@ public void updateTask(TaskResult taskResult, String taskType) {
                 taskResult.setReasonForIncompletion(String.format("The TaskResult payload: %d is greater than the permissible 3MB", taskResultSize));
                 taskResult.setStatus(TaskResult.Status.FAILED_WITH_TERMINAL_ERROR);
                 taskResult.setOutputData(null);
+                taskResult.setOutputMessage(null);
```

Comment:
please also update the logic that limit size of workflow input, task output and task result to consider size of inputMessage and outputMessage as well.

Reply:
@cyzhao: Since we're passing `taskResult` to the `countingOutputStream`, the size of the new `Any` fields is already being taken into account, as the `ObjectMapper` instance which we're injecting takes care of serializing those two fields.